modules/node-shared/src/main/resources/application.conf

```diff
@@ -146,4 +146,10 @@ validation-error-storage {
 delegated-staking {
   min-reward-fraction:  5000000
   max-reward-fraction: 10000000
+  # (10 years * 365.25 days * 24 hours * 60 minutes * 60 seconds) / 43 seconds per epoch
+  # in epochs
+  min-epoch-progresses-to-lock: 7338977
+  # 21 days * 24 hours * 60 minutes * 4.5 snapshots per minute
+  # in snapshot ordinals
+  withdrawal-time-limit = 136080
```

Comment:
Since we'll need another configuration specific to testing (haven't reviewed full PR so if this has already been done please ignore this comment.) But -- we need another conf / or some way to introduce acceptable parameters for testing this, i.e. a very small amount.